ci/slab.toml

```diff
@@ -56,6 +64,11 @@ workflow = "aws_tfhe_gpu_tests.yml"
 profile = "gpu-test"
 check_run_name = "GPU AWS Tests"
 
+[command.multi_gpu_test]
+workflow = "aws_tfhe_gpu_tests.yml"
+profile = "multi-gpu-test"
+check_run_name = "Multi GPU AWS Tests"
+
```

Comment:
Using this won't work anymore since we spawn the instance directly in the workflow. Instead we should do a kind of matrix build with the single GPU profile and the multi one.